docs/UserGuide.md

```diff
@@ -1,134 +1,162 @@
 # User Guide
-
 * [Quick Start](#quick-start)
 * [Features](#features)
 * [FAQ](#faq)
 * [Command Summary](#command-summary)
 
 ## Quick Start
 
-0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
+Step 1: Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
    > Having any Java 8 version is not enough. <br>
-   This app will not work with earlier versions of Java 8.
+   This application will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+Step 2: Download the latest `taskell.jar` from here.<br>
+	<br> <img src="images/Icon.png" width="100"></br>
+Step 3: Copy the file to the folder you want to use as the home folder for your Task Manager.<br>
+Step 4: Double-click the file to start the application. The GUI should appear in a few seconds. <br>
+ <br><img src="images/GUI.png" width="600"><br>
```

Comment:
the GUI mockup shown here is quite a deviation from the addressbook-level4 GUI, and looks quite basic; while this is acceptable, remember that you can reuse the GUI code from AB-L4 for much of the things. 
it is ok to remove the browser component totally, but do think of making use of the available space effectively to display useful information to the user. 
during the course of the project we can see how you can make use of GUI components to give good feedback to the user when specific commands are executed.